product_available_by_warehouse/models/product.py

```diff
@@ -40,3 +40,11 @@ def get_product_available_by_warehouse(self):
                 'outgoing_qty': outgoing_qty,
             }
         return res
+
+    @api.multi
+    def _get_domain_locations(self):
```

Comment:
@ysantiago 

I do not know if is the correct make this super, it suppose when you call the product_available return quantity only of internal location and you do not need modify into native method in odoo like this,

@suniagajose 
what do you think?